src/qml/QFieldCloudScreen.qml

```diff
@@ -180,6 +180,16 @@ Page {
         }
       }
 
+      SearchBar {
+        Layout.fillWidth: true
+        Layout.preferredHeight: 41
+        color: Theme.mainBackgroundColor
+
+        onValidateFilter: function (filter) {
+          table.model.setTextFilter(filter);
+        }
```

Comment:
See comment above, it'd become table.model.textFilter = filter 

But also, I'm suggesting below to get rid of the signal in favor of a searchTerm property (and get rid of the need to hit return to search something). 

If somehow reacting to every k entered is too hard of the model, we can use a timer to set the text filter after a number of milliseconds.